docs/index.md

```diff
@@ -0,0 +1,33 @@
+# Synapse Operator
+
+A Juju charm deploying and managing [Synapse](https://github.com/matrix-org/synapse) on Kubernetes.
+Synapse is a drop in replacement for other chat servers like Mattermost and
+Slack.
+
+This charm simplifies initial deployment and “day N” operations of Synapse on
+Kubernetes, such as integration with SSO, access to S3 for redundant file
+storage and more. It allows for deployment on many different Kubernetes
+platforms, from [MicroK8s](https://microk8s.io) to [Charmed Kubernetes](https://ubuntu.com/kubernetes)
+to public cloud Kubernetes offerings.
+
+As such, the charm makes it easy for those looking to take control of their own
+Chat server whilst keeping operations simple, and gives them the freedom to
+deploy on the Kubernetes platform of their choice.
+
+For DevOps or SRE teams this charm will make operating Synapse simple and
+straightforward through Juju’s clean interface. It will allow easy deployment
+into multiple environments for testing of changes.
+
+## Project and community
```

Comment:
This section is not needed anymore here and we lack "Contributing to this documentation" and "In this documentation" sections

Reply:
Related: #50